File: competidores/P12/maior.java
import java.util.Scanner;

public class maior {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        
        int n1, n2, n3, n4;
        char op1, op2;
        double exp1=0, exp2=0;
        
        n1 = in.nextInt();
        op1 = in.next().charAt(0);
        n2 = in.nextInt();
        n3 = in.nextInt();
        op2 = in.next().charAt(0);
        n4 = in.nextInt();
        
        if(op1 == '+'){
            exp1 = n1 + n2;
        }
        else if(op1 == 'x'){
            exp1 = n1 * n2;
        }
        else if(op1 == '^'){
            exp1 = Math.pow(n1, n2);
        }
        if(op2 == '+'){
            exp2 = n3 + n4;
        }
        else if(op2 == 'x'){
            exp2 = n3 * n4;
        }
        else if(op2 == '^'){
            exp2 = Math.pow(n3, n4);
        }
        if(exp1 > exp2){
            System.out.println(">");
        }
        else if(exp1 < exp2){
            System.out.println("<");
        }
        else{
            System.out.println("=");
        }
    }
    
}

File: competidores/P10/maior.c
#include <stdio.h>
#include <math.h>

int main(){
	
	int n1, n2, n3, n4, res1, res2,i=1;
	char op1, op2;	
		
	scanf("%d%c%d %d%c%d", &n1, &op1, &n2, &n3, &op2, &n4);
		
	if(op1 == '+') res1 = n1+n2;
	
	if(op1 == 'x') res1 = n1*n2;		
	
	if(op1 == '^') {
	
		res1 = n1;
		
		while(i < n2){
			res1 = res1*n1;	
			i++;
		}
		i=1;
	}
		
	if(op2 == '+') res2 = n3+n4;
		
	if(op2 == 'x') res2 = n3*n4;		
	
	if(op2 == '^') {
	
		res2 = n3;
		
		while(i < n4){
			res2 = res2*n3;	
			i++;
		}
		i=1;
	}
	
	if(res1 > res2){
		printf(">");
	}

	if(res1 = res2){
		printf("=");
	}
	
	if(res1 < res2){
		printf("<");
	}
	
	//printf("%d %d", res1, res2);
	
	return 0;

}
